src/ui/src/builder/useComponentActions.ts

```diff
@@ -127,6 +127,7 @@ export function useComponentActions(wf: Core, ssbm: BuilderManager) {
 		type: string,
 		parentId: Component["id"],
 		position?: number,
+		initProperties?: Partial<Component>,
```

Comment:
Since many props are overridden

https://github.com/writer/writer-framework/blob/f17a349a11f7df87de53dc04227fbccd40b01db8/src/ui/src/builder/useComponentActions.ts#L140-L148

the type could be a bit more correct
```suggestion
		initProperties?: Partial<Omit<Component, 'id' | 'type' | 'parent' | 'content' | 'handlers' | 'position'>>,
```

Reply:
It does make sense, yes. Thanks.